docs/opentracing.asciidoc

```diff
@@ -111,31 +111,31 @@ Elastic APM defines some tags which are not included in the OpenTracing API but
 
 [float]
 [[unsupported]]
-=== Caveats
+==== Caveats
 Not all features of the OpenTracing API are supported.
 
 [float]
 [[propagation]]
-==== Context propagation
+===== Context propagation
 This bridge only supports the formats `Format.Builtin.TEXT_MAP` and `Format.Builtin.HTTP_HEADERS`.
 `Format.Builtin.BINARY` is currently not supported.
 
 [float]
 [[references]]
-==== Span References
+===== Span References
 Currently, this bridge only supports `child_of` references.
 Other references,
 like `follows_from` are not supported yet.
```

Comment:
[question] would it be relevant to make the agent issue a warning once whenever this feature is used for the 1st time ?
Same for Baggage below. So in order to know if features would be missing vs regular OTel implementation we just have to scan the logs for any warning.

Reply:
Done